src/z3c/testing/app.py

```diff
@@ -11,10 +11,9 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""
-$Id$
-"""
-
+from zope.container.tests.test_icontainer import BaseTestIContainer as BTIC
+from zope.container.tests.test_icontainer import DefaultTestData
+from zope.interface.verify import verifyObject, verifyClass
 import unittest
```

Comment:
This is the part of PEP-8 I do not like. In my projects I have only one block containing all the import statements alphabetically sorted, so `from` imports are above all other ones. (Interestingly the PEP-8 checker  does not complain about miss ordered imports.)

But as it seems to be consensus to follow this part of PEP-8 in the zope projects, keep nagging me if I forget it again.